lib/src/model/analysis/retro_controller.dart

```diff
@@ -85,16 +80,12 @@ class RetroController extends _$RetroController with EngineEvaluationMixin {
 
   late ExportedGame _game;
 
+  final Completer<void> _serverAnalysisCompleter = Completer<void>();
+
   @override
   @protected
   EngineEvaluationPrefState get evaluationPrefs =>
-      state.valueOrNull?.feedback == RetroFeedback.evalMove
-      // Increase search time to make sure we reach the limit for deciding whether the move
-      // is good enough to be considered an alternative solution.
-      ? ref
-            .read(engineEvaluationPreferencesProvider)
-            .copyWith(engineSearchTime: kMaxEngineSearchTime)
-      : ref.read(engineEvaluationPreferencesProvider);
+      ref.read(engineEvaluationPreferencesProvider).copyWith(isEnabled: true, numEvalLines: 1);
```

Comment:
These settings should not change based on some logic, as this will effectively kill and restart the engine.

Reply:
Makes sense, maybe we can add a warning comment about this somewhere in the code?